src/Cafe/HW/Latte/Renderer/Vulkan/VulkanPipelineCompiler.cpp

```diff
@@ -459,8 +459,7 @@ void PipelineCompiler::InitVertexInputState(const LatteContextRegister& latteReg
 		VkVertexInputBindingDescription entry{};
 #if BOOST_OS_MACOS
 		if (bufferStride % 4 != 0) {
-			forceLog_printf("MoltenVK error: vertex stride was %d, expected multiple of 4", bufferStride);
-			bufferStride = 0;
+			bufferStride = bufferStride + (4-(bufferStride % 4));
```

Comment:
The statement only runs when it isn't a multiple of 4

Reply:
Oh right. I didn't look at the context. You can leave it as-is then. Although personally I prefer making code foolproof in case the surrounding logic ever changes.